sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_client_async.py

```diff
@@ -741,6 +749,7 @@ async def _message_received_async(self, frame, message):
         :param message: Received message.
         :type message: ~pyamqp.message.Message
         """
+        self._last_activity_timestamp = time.time()
```

Comment:
on_receive on a message means there was activity -- this is used to measure timeout